src/Controls/src/Core/Platform/AlertManager/AlertManager.Windows.cs

```diff
@@ -209,7 +209,7 @@ void OnActionSheetRequested(Page sender, ActionSheetArguments arguments)
 
 				try
 				{
-					var current = sender.ToPlatform(VirtualView.RequireMauiContext());
+					var current = sender.ToPlatform(sender.RequireMauiContext());
```

Comment:
can this just be `sender.ToPlatform()`

At this point the sender should already have handler and all we're wanting to do here is retrieve it